### PR TITLE
Add NW benchmark smoke test

### PR DESCRIPTION
**Pull request by arun-koshy**
```
fab smoke  11.86s user 2.39s system 99% cpu 14.308 total
```

### CODE REVIEW

File: .github/workflows/rust.yml [hunk 1]
```diff
@@ -78,6 +78,7 @@ jobs:
       - name: benchmark (smoke)
         run: |
           cargo run --package sui-benchmark --bin stress -- --log-path /tmp/stress.log --num-client-threads 10 --num-server-threads 24 --num-transfer-accounts 2 bench --target-qps 100 --num-workers 10  --transfer-object 50 --shared-counter 50 --run-duration 10s
+          pushd narwhal/benchmark && fab local && popd
```
Comment (mwtian) [inline .github/workflows/rust.yml:81]: Should this be `fab smoke` or something that uses the new mode?

File: .github/workflows/rust.yml [hunk 2]
```diff
@@ -36,10 +36,10 @@ jobs:
     outputs:
       isRust: ${{ steps.diff.outputs.isRust }}
     steps:
-    - uses: actions/checkout@v3
-    - name: Detect Changes
-      uses: './.github/actions/diffs'
-      id: diff
+      - uses: actions/checkout@v3
```
Comment (mwtian) [inline .github/workflows/rust.yml:39]: Just curious, what is causing the auto formatting?

**Review by mwtian — COMMENTED** [review]
Seems we need to install fab as well.

File: .github/workflows/rust.yml (hunk 1 above)
Comment (arun-koshy) [reply .github/workflows/rust.yml]: Thanks for catching this! Should be `smoke` 🤦🏽

File: .github/workflows/rust.yml (hunk 2 above)
Comment (arun-koshy) [reply .github/workflows/rust.yml:39]: I have a few formatter extensions on vscode but I believe `Prettier` is the one that auto formatted this file.

**Review by mwtian — APPROVED** [review]
Great, there is not much increase on the smoke test duration.

**Pull request merged**